modules/configure-signature-integration.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * operating/verify-image-signatures.adoc
+:_module-type: PROCEDURE
+[id="configure-signature-integration_{context}"]
+= Configuring signature integration
+
+Before performing image signature verification, you must first add your Cosign public keys in {product-title-short}.
+
+.Prerequisites
+
+* You must already have a PEM-encoded public key.
```

Comment:
Maybe it makes sense to also link here to [the quick start of cosign](https://github.com/sigstore/cosign#quick-start), so people unfamiliar with it can solve the prerequisites right away?